From 67c016deba56b7dc665cd3bd82c0027f760fe5e6 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Tue, 26 Sep 2017 18:41:46 -0500
Subject: [PATCH 252/278] drivers/pci/host: Fix Axxia Compile Errors

The MSI changes to the Axxia driver should only be
included when CONFIG_PCI_MSI is defined.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 drivers/pci/host/pcie-axxia.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/pci/host/pcie-axxia.c b/drivers/pci/host/pcie-axxia.c
index 5f8248a..3862818 100644
--- a/drivers/pci/host/pcie-axxia.c
+++ b/drivers/pci/host/pcie-axxia.c
@@ -32,7 +32,9 @@
 
 #include "pcie-axxia.h"
 
+#ifdef CONFIG_PCI_MSI
 #define AXXIA_GENERIC_MSI_DOMAIN_IRQ 1
+#endif	/* CONFIG_PCI_MSI */
 
 #define PEI_GENERAL_CORE_CTL_REG 0x38
 #define PEI_SII_PWR_MGMT_REG 0xD4
-- 
1.8.5.2.233.g932f7e4

